guides/common/modules/proc_synchronizing-the-project-client-name-repository-for-rhel-9-and-rhel-8.adoc

```diff
@@ -0,0 +1,41 @@
+[id="synchronizing-the-project-client-name-repository-rhel-9-and-rhel-8"]
+= Synchronizing the {project-client-name} repository for {RHEL} 9 and {RHEL} 8
+
+To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for your {RHEL} version:
+
+* xref:CLI_Synchronizing_the_Client_Repository_rhel_9_{context}[]
+* xref:CLI_Synchronizing_the_Client_Repository_rhel_8_{context}[]
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Sync Status*.
+. Click the arrow next to the *{RHEL} for x86_64* product to view available content.
+. Select *{Team} {project-client-name} for RHEL 9 x86_64 RPMs* or *{Team} {project-client-name} for RHEL 8 x86_64 RPMs*.
+. Click *Synchronize Now*.
+
+[id="CLI_Synchronizing_the_Client_Repository_rhel_9_{context}"]
```

Comment:
We can use sentence case here as well?

Reply:
Currently, we either use camel case or lowercase anchors, I would rather not introduce another type. That being said, RHEL in this anchor could be capitalized, but if it is an anchor that already existed, I'd probably rather opt for not touching it.

Reply:
I chose not to update the anchors because as Adam wrote, they were already in place before. To be honest, I don't know if changing only capitalization would make existing links invalid.

Does anyone else have any thoughts on it? Is it safe to update them to lowercase?

Reply:
Well, if you wanted to go with lowercase anchor in the format that we use, you would also have to replaces underscores with dashes, which then would make the existing links invalid. It is not problematic to update the links within our documentation, but I simply do not see the added value.

Reply:
I am sorry, I meant lowercase (as we are working to convert Title Case in IDs). Agree to Adam's view.

Reply:
Let's keep the IDs the way they are then. Thanks everyone!